bundle-workflow/src/run_integ_test.py

```diff
@@ -57,13 +59,24 @@ def main():
                     work_dir,
                     args.s3_bucket
                 )
-                test_suite.execute()
+                status = test_suite.execute()
+                if status != 0:
+                    failed_components[component.name] = status
+                else:
+                    passed_components[component.name] = status
             else:
                 logging.info(
                     "Skipping tests for %s, as it is currently not supported"
                     % component.name
                 )
 
+        if failed_components:
+            for component, status in failed_components.items():
+                logging.error(f'FAIL: Integration Test for {component} with status code {status}')
+        if passed_components:
+            for component, status in passed_components.items():
+                logging.info(f'PASS: Integration Test for {component} with status code {status}')
+
```

Comment:
Also after printing out the summary, if there are failed components make sure the `run_integ_test.py` also fails with non-zero status code.